Libs/MRML/DisplayableManager/vtkMRMLModelDisplayableManager.cxx

```diff
@@ -785,8 +785,9 @@ bool vtkMRMLModelDisplayableManager::IsModelDisplayable(vtkMRMLDisplayableNode*
   {
     displayable |= this->IsModelDisplayable(node->GetNthDisplayNode(i));
     if (displayable)
-    {// Optimization: no need to search any further.
-      continue;
+    {
+      // Optimization: no need to search any further.
+      break;
```

Comment:
Fixed an issue that I just noticed in IsModelDisplayable. Should be break not continue.